app/views/supervisors/index.html.erb

```diff
@@ -56,3 +56,24 @@
     </table>
   </div>
 </div>
+
+
+
+<table class="table">
+    <thead>
+      <tr>
+        <th>Volunteers not assigned to supervisors</th>
+      </tr>
+    </thead>
+      <tbody>
+        <td>
+          <% if @available_volunteers.count <= 0 %>
+            <%= "Currently no volunteers are unassigned"%>
+          <% else %>
+            <%  @available_volunteers.active.each do |volunteer| %><br>
```

Comment:
Unassigned volunteers:

Reply:
from office hours 

```

  context "with unassigned volunteers" do
    let!(:unassigned_volunteer) { create(:volunteer, casa_org: organization, display_name: "Tony Ruiz") }

    before do
      sign_in user
      visit supervisors_path
    end

    it "will show a list of unassigned volunteers" do
      expect(page).to have_text("Unassigned volunteers:")
      expect(page).to have_text("Tony Ruiz") # test link too
      expect(page).not_to have_text("Currently no volunteers are unassigned")
    end
  end

  context "without unassigned volunteers" do
    before do
      sign_in user
      visit supervisors_path
    end
    it "will not show a list of volunteers not assigned to supervisors" do
      expect(page).to have_text("Currently no volunteers are unassigned")
    end
  end
end
```

Reply:
I assigned this to you :) https://github.com/rubyforgood/casa/issues/1990

Reply:
Thank you @compwron